components/zcash_address/src/kind/unified.rs

```diff
@@ -186,7 +186,7 @@ pub(crate) mod private {
         }
 
         fn write_raw_encoding<W: Write>(&self, mut writer: W) {
-            for item in &self.items() {
+            for item in self.items_as_parsed() {
```

Comment:
This is okay in itself, but we are not conforming to the SHOULD requirement for the ordering of items in [ZIP 316](https://zips.z.cash/zip-0316#requirements-for-both-unified-addresses-and-unified-viewing-keys).

Reply:
Fixed in d307ee8d8b5aa3d23c750953c0fe4b2f770e86df